module/userExtension/helper.js

```diff
@@ -0,0 +1,242 @@
+
+let sunBirdService =
```

Comment:
@deepakr123 , @rakeshSgr - Please change it to sunbirdService. Its once word and hence no camel case between sun. and bird

Reply:
resolved